cd/linux-packages/yum/descriptor.json

```diff
@@ -0,0 +1,22 @@
+{
+    "package": {
+        "name": "Yum",
+        "repo": "apache-arrow-yum",
+        "subject": "kou",
+        "licenses": ["Apache-2.0"],
+        "vcs_url": "htttps://github.com/apache/arrow.git"
+    },
+    "version": {
+        "name": "0.9.0"
```

Comment:
This will need to be templated or otherwise configuable. We hit this during the 0.9.0 release.

Reply:
Templating currently only works for the CI configurations, so we should do that in the rakefile itself.

Reply:
It's just a label for version. e.g.: https://bintray.com/kou/apache-arrow-yum/Yum/0.9.0
We can use "dev" or something statically for it.